src/status_im/ui/screens/screens.cljs

```diff
@@ -219,6 +220,9 @@
     :options   {:insets {:top? true}}
     :component wallet.swap/asset-selector}
 
+   {:name      :address-to-watch-edit
```

Comment:
@tumanov-alex this should be in status_im2 screens file. please move there before merging 👍